.circleci/config.yml

```diff
@@ -209,6 +209,7 @@ workflows:
             ((.circleci|jupyter-extension)/.*) run-jupyter-jobs true
             ((.circleci|python-sdk)/.*) run-python-sdk-jobs true
             ((.circleci|label-studio)/.*) run-label-studio-jobs true
+            ((.circleci|console)/.*) run-console-jobs true
```

Comment:
I am not 100% sure I set this up correctly. I need to investigate a little further here.

Reply:
This seems correct to me. The reason why all of the test suites are running for this PR is because all of the test suites depend on the `.circleci` directory (which is kind of annoying but I'm not sure there's a better way).